src/inmanta/server/services/orchestrationservice.py

```diff
@@ -560,12 +568,16 @@ def _create_dao_resources_from_api_resources(
         resource_state: dict[ResourceIdStr, Literal[ResourceState.available, ResourceState.undefined]],
         resource_sets: dict[ResourceIdStr, Optional[str]],
         set_version: Optional[int] = None,
-    ) -> dict[ResourceIdStr, data.Resource]:
+    ) -> tuple[dict[ResourceIdStr, data.Resource], dict[ResourceIdStr, data.ResourceSet]]:
```

Comment:
Note to self: review this method, depending on how the other discussion evolves.